README.md

```diff
@@ -1 +1,43 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## **About Me**
+
+![*A picture of Berenice*](Bere.jpg)
```

Comment:
Great job inserting the image! I personally struggled with image insertion when completing my assignment, but it is nice to see you were able to make this photo part of your introduction.

Reply:
Yes! nice way to start. I also struggled to upload images. 
Next time I'll check the "Mastering Markdown guide" that you mentioned.